api-docs/tech-ref-info/check-types/agent-check-type-ref.rst

```diff
@@ -188,8 +188,8 @@ Metrics
 +-----------------+--------------------------------------------------+----------+
 | Metric          | Description                                      | Type     |
 +=================+==================================================+==========+
-| avail           | Available space on the filesystem in kilobytes,  | Int64    |
-|                 | including reserved space.                        |          |
```

Comment:
Hi, @wolfdancer: That's what I assumed. Might be a non-issue. Let me know if you hear back from the engineers.